### DISCUSSION TITLE
Enhancement: "pushable" chapters

### DISCUSSION THREAD

**Discussion by theDanielJLewis** [Podcastindex-org/podcast-namespace#387] [Ideas]
We can already do a lot with chapters (and maybe more in the future). What if podcasters could make a chapter push a notification to the listener's device?

Imagine this. You're listening to a podcast and they mention an important link. Instead of rushing to unlock your device, open the podcast app, and tap the chapter link before it goes away, your podcast app could push that chapter as a notification so that simply tapping the notification will go to the thing you want.

Podcasters could use this for important calls to action and future interactions we may someday build into chapters (like polls, voting, feedback, etc.).

The spec could be as simple as a `push: true` field in the chapter JSON. Then, apps could offer that as an additional notification option in their apps, along with "notify of new episodes" and "notify when live."

What do you think?

**Comment by brianoflondon** [comment]
I believe group editable and hosted chapters could be stored in Hive. 

It's not something I have time to build but I if anyone wants a project I can give direction and I dare say even get it funded after a basic prototype. 

The reason I suggest Hive is that it is an open public database that any app can read and write to. There's nothing else like it. It has all the necessary permission and identity stuff already built.

**Reply by theDanielJLewis** [reply]
Wouldn't that be horribly slow for multimedia content, like a 1 MB file?

**Reply by brianoflondon** [reply]
Chapters only point to graphics. 

Go to https://peakd.com/hive-110369/@brianoflondon/for-the-bitcoin-bros-who-call-everything-they-wont-look-at-or-understand-a-shit-coin

The text for that page is stored in Hive but delivered via dedicated API servers which are pulling from a regular database. 

Now compare these links. Completely different front ends same back end. 

https://hivel.ink/hive-110369/@brianoflondon/for-the-bitcoin-bros-who-call-everything-they-wont-look-at-or-understand-a-shit-coin
